testapi.pm

```diff
@@ -712,7 +714,9 @@ Run C<$command> via C<script_sudo> and then check by C<wait_serial> if its exit
 status is not zero.
 See C<wait_serial> for default timeout.
 
-I<The implementation is distribution specific and not always available.>
+I<The implementation is distribution specific and not always available. Note this
+is quite likely to fail with the default C<script_sudo> implementation, as it attempts
+to echo to the serial device without root permissions.>
```

Comment:
IMHO not true in general.

If it would be true, the notice should be on `script_run` as well.

That is depending on how the tests are implemented and how the distribution handles permissions of the serial device, whatever has been provided as a serial device. fedora has the same default permission for /dev/ttyS0 as *suse but that does not mean it can not work. Either the test non-root user is added to the group, permissions are adjusted, owner is changed (how *suse does it: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/consoletest_setup.pm#L73 - yes, not necessarily nice), a special serial device is given reserved for that use.

I suggest:

> Make sure the non-root user has permissions to write to the supplied serial device C<$serialdev>